playground.js

```diff
@@ -14,10 +14,11 @@
 // limitations under the License.
 
 import * as Schism from  './rt/rt.js';
+import filesystem from './rt/filesystem-null.js';
 
 async function compileSchism() {
   const schism_bytes = await fetch('schism-stage0.wasm', { credentials: 'include' });
-  const engine = new Schism.Engine;
+  const engine = new Schism.Engine(filesystem);
```

Comment:
If there are any other configuration options that we can consider adding, then perhaps we should pass this as an options dict -- i.e. `{filesystem}`.  Just a thought.

Reply:
Good idea. I opened #88 to remember this, and hopefully I'll get to it relatively soon (unless someone else gets there first). I'll go ahead and merge this PR as is though.

Another use for configuration options could be disabling certain Wasm features. For example, we could do feature detection to see if tail calls are supported, and generate regular calls if they aren't.